### PR TITLE
Center info page

### PR DESCRIPTION
**Pull request by thebagmaster**
The design file was updated to show the cover for the onboarding prompt was changed to be centered. This required some cleanup in both in InformationPage as well as OnboardingCover. This small change should help with future usages of InformationPage.

The changes include:
- ~Removing states from InformationPage in favor of local bool~ Just some edits to the changes for the states
- Adding an encompassing ColumnLayout as well as a ~filler item~ < Bad practice
- Moving control for centering into OnboardingCover

`edit: see comments below for working screenshots.`

ss

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/unsecure_win_gui.zip?branch=pull/345)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/unsecure_mac_gui.zip?branch=pull/345)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/unsecure_mac_arm64_gui.zip?branch=pull/345)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/unsecure_android_apk.zip?branch=pull/345)
[ARM32 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android32/unsecure_android_32bit_apk.zip?branch=pull/345)

### CODE REVIEW

File: src/qml/pages/onboarding/OnboardingCover.qml [hunk 1]
```diff
@@ -48,6 +51,7 @@ Page {
             descriptionSize: 24
             subtext: qsTr("100% open-source & open-design")
             buttonText: qsTr("Start")
+            centerContent: AppMode.state=="MOBILE"
```
Comment (jarolrod) [inline src/qml/pages/onboarding/OnboardingCover.qml:54]: this can be AppMode.isDesktop instead

**Review by jarolrod — COMMENTED** [review]
from linter: 

```
This diff appears to have added new lines with trailing whitespace.
The following changes were suspected:

diff --git a/src/qml/controls/InformationPage.qml b/src/qml/controls/InformationPage.qml
@@ -52 +53,6 @@ Page {
+        
@@ -93,0 +97,5 @@ Page {
+            Item { 
+              Layout.fillHeight: !root.centerContent 
```

File: src/qml/controls/InformationPage.qml [hunk 2]
```diff
@@ -102,46 +110,9 @@ Page {
...
+            anchors.bottom: parent.bottom
             text: root.buttonText
             onClicked: root.lastPage ? swipeView.finished = true : swipeView.incrementCurrentIndex()
         }
     }
-
-    state: AppMode.state
-
-    states: [
-        State {
```
Comment (johnny9) [inline src/qml/controls/InformationPage.qml:113]: The states are required to handle the anchoring differences between mobile and desktop. anchors changes should be handled through States/AnchorChanges. https://doc.qt.io/qt-6/qtquick-positioning-anchors.html#changing-anchors)](https://doc.qt.io/qt-6/qtquick-positioning-anchors.html#changing-anchors

File: src/qml/controls/InformationPage.qml [hunk 3]
```diff
@@ -83,14 +87,18 @@ Page {
...
                 visible: active
-                Layout.fillWidth: true
                 Layout.alignment: Qt.AlignCenter
                 Layout.topMargin: root.detailTopMargin
                 Layout.leftMargin: 20
                 Layout.rightMargin: 20
                 Layout.maximumWidth: detailMaximumWidth
                 sourceComponent: root.detailItem
             }
+            Item {
```
Comment (johnny9) [inline src/qml/controls/InformationPage.qml:97]: Use AnchorChanges instead of adding a useless Item.

**Comment by thebagmaster** [comment]
Rebased
Added the states back in
Had to respect button position for desktop should behave like design doc again
SS1
SS2

**Comment by jarolrod** [comment]
Closing for now; this needs to be rebased over main, but we'll be taking a different approach as to the view components and how we handle desktop vs mobile

**Pull request closed without merging**